generators/pythonv2/codegen/src/ast/Variable.ts

```diff
@@ -0,0 +1,37 @@
+import { AstNode } from "./core/AstNode";
+import { Writer } from "./core/Writer";
+import { Annotation } from "./Annotation";
+
+export declare namespace Variable {
+    interface Args {
+        /* The name of the variable */
+        name: string;
+        /* The type annotation of the variable */
+        type?: Annotation;
```

Comment:
thoughts on this just being a `Type.ts` -- forces someone to create a `Type` when instantiating a `Field`